test/run

```diff
@@ -385,6 +385,7 @@ hardlink
 direct
 direct_gcc
 depend
+depend_different_headers
```

Comment:
No need to create a new suite containing only one test. I suggest adding the test case to the depend suite instead.

Reply:
I wanted to do that, but the suite setup for that one test and also the test itself is too long and elaborate to fit in there.